api-service/api-server/src/setup/questions_jsonlines_transformer.ts

```diff
@@ -85,7 +85,7 @@ async function main(args: Params) {
 
   let interpretStream = new multiplex_transform.Multiplex();
   interpretStream.setEncoding('utf-8');
-  interpretStream.setInputProcessor((chunk, encoding, pushFn) => {
+  interpretStream.setInputProcessor((chunk:string, _encoding:string, pushFn:Function) => {
```

Comment:
Would it be possible to more strongly type this Function argument? Also, style nit: Would prefer a space after the ":".